examples/express-graphql-http-functions/Query.ts

```diff
@@ -0,0 +1,73 @@
+import { Context } from "./context";
+import User from "./models/User";
+
+/**
+ * @gqlType
+ */
+export type Query = unknown;
+
+
+/**
+ * @gqlField users
+ */
+export function usersQueries(_: Query): UserQueries {
+  return {}
+}
+
+/**
+ * @gqlType
+ */
+export type UserQueries = {};
```

Comment:
Might be a good place to use docblock descriptions to clarify what this field/type is demonstrating.